e2e_tests/tests/experiment/test_delete.py

```diff
@@ -0,0 +1,40 @@
+import pathlib
+import subprocess
+import time
+
+import pytest
+
+from determined.common import api
+from tests import config as conf
+from tests import experiment as exp
+
+
+@pytest.mark.e2e_cpu
+def test_delete_experiment_removes_tensorboard_files() -> None:
+    """
+    Start a random experiment, delete the experiment and verify that TensorBoard files are deleted.
+    """
+    config_obj = conf.load_config(conf.tutorials_path("mnist_pytorch/const.yaml"))
+    experiment_id = exp.run_basic_test_with_temp_config(
```

Comment:
e2e tests are expensive. if this test just needs to check for presence and deletion of tensorboard files, can we run an experiment that has minimal overhead? like `no_op/single-one-short-step.yaml`.

Reply:
We need to have an experiment that generates TB files. I am using `single-medium-train-step.yaml` which might have slightly less overhead